lib/puma/cluster.rb

```diff
@@ -95,12 +95,13 @@ def initialize(idx, pid, phase, options)
         @signal = "TERM"
         @options = options
         @first_term_sent = nil
+        @started_at = Time.now
```

Comment:
cluster inherits from runner, so I believe this is already initialized, no?

Reply:
But this is for the `Worker` class as you can see L89